authentic/authentic.config.py

```diff
@@ -84,11 +88,18 @@
             'class': 'logging.StreamHandler',
             'formatter': 'simple'
         },
+        'file': {
+            'level': 'DEBUG',
+            'class': 'logging.FileHandler',
+            'filters': ['require_debug_true'],
+            'filename': '/var/log/authentic2-multitenant/django.log',
```

Comment:
Logs are available in systemd... adding a file log in production must be an option as a log rotate mecanism should also be configured (already available in systemd)

Reply:
I just pushed in this branch an initial implementation for this